### DISCUSSION TITLE
Send a file blob to the front-end

### DISCUSSION THREAD

**Discussion by hpharmsen** [reflex-dev/reflex#3191] [Q&A] [answered]
My back-end generates a temporary (xml) file which I want to offer to the user as a download.

In JavaScript I would do:
`let blob = new Blob([myXml], {type: 'application/xml'}); `
`let url = URL.createObjectURL(blob);`
  
  How can I do this in Reflex?

**Comment by masenf** [comment, marked as answer]
See examples here: https://reflex.dev/docs/assets/upload-and-download-files/#with-

You can pass `str` or `bytes` to the `rx.download` special event to trigger a download of temporary data.
